fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1467,7 +1494,7 @@ Reference<ClientInfo> MultiVersionApi::getLocalClient() {
 
 void MultiVersionApi::selectApiVersion(int apiVersion) {
 	if (!localClient) {
-		localClient = makeReference<ClientInfo>(ThreadSafeApi::api);
+		//		localClient = makeReference<ClientInfo>(ThreadSafeApi::api);
```

Comment:
If we're getting rid of this, we can get rid of the whole if block.

Reply:
Oh, I forgot to change this. It should take IClientApi::localApi here